setup.py

```diff
@@ -10,7 +10,7 @@
     version=__version__,
     author='Pierre Nicolas Durette',
     author_email='pndurette@gmail.com',
-    url='https://github.com/pndurette/gTTS',
+    url='https://github.com/dixya/gTTS',
```

Comment:
Haha, since this is a pull request you should not change the main repo url with our fork version.